Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docrepository/adapter/model/DocumentMetadata.java

```diff
@@ -538,6 +541,9 @@ public String getNewRepositoryUniqueId() {
     }
 
     public void setNewRepositoryUniqueId(String newRepositoryUniqueId) {
+        if (null != document && null == document.getRepositoryUniqueId()) {
```

Comment:
Do we need to check if newRepositoryUniqueId is not null here?

Reply:
the Repository-Unique-id is checked in the business-layer on the AdminGUI

Reply:
currently the NewRepositoryUniqueId and RepositoryUniqueId are different when we first-initialize the Repository-Unique-id they are the same; newRepositoryUnuiqueId is part of the MetaData, and RepositoryUniqueId is part of document